vimrc.bundles

```diff
@@ -2,6 +2,14 @@ if &compatible
   set nocompatible
 end
 
+" Remove declared plugins
+function! s:UnPlug(plug_name)
+  if has_key(g:plugs, a:plug_name)
```

Comment:
Following on with my comment about re: `UnPlug` needing to work with just the pluging name, not `author/plugin-name`, do we think it would be feasible / useful to either warn or handle the case where a user tries to unplug with `author/plugin-name` argument?

``` vim
if match(a:plug_name, '\v^[^\/]+\/[^\/]+$')
  let plug_name_without_author = split(a:plug_name, '/')[1]
endif
```